curvefs/src/metaserver/metastore_fstream.cpp

```diff
@@ -331,7 +353,24 @@ bool MetaStoreFStream::Load(const std::string& pathname) {
         return false;
     };
 
-    return LoadFromFile(pathname, callback);
+    auto ret = LoadFromFile(pathname, callback);
+
+    std::ostringstream oss;
+    oss << "total partition: " << totalPartition
+        << ", total inode: " << totalInode << ", total dentry: " << totalDentry
+        << ", total s3chunkinfolist: " << totalS3ChunkInfoList;
+
+    if (ret) {
+        LOG(INFO) << "Metastore "
+                  << copyset::ToGroupIdString(poolId_, copysetId_)
+                  << " load from " << pathname << " succeeded, " << oss.str();
+    } else {
+        LOG(WARNING) << "Metastore "
```

Comment:
Will retry when failed?

Reply:
noop, it just reports an error that will cause copyset error.

the log level has changed to ERROR